ci/v2/test/t.py

```diff
@@ -0,0 +1,78 @@
+import click
+import os
+import subprocess
+from typing import List
+
+import numpy as np
+
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+
+
+@click.command()
+@click.argument("targets", required=True, type=str)
+@click.argument("team", required=True, type=str)
+@click.option(
+    "--concurrency",
+    default=3,
+    type=int,
+    help=("Number of concurrent test jobs to run."),
+)
+@click.option(
+    "--shard",
+    default=0,
+    type=int,
+    help=("Index of the concurrent shard to run."),
+)
+def main(targets: str, team: str, concurrency: int, shard: int) -> None:
+    test_targets = get_test_targets(targets, team)
+    run_tests(np.array_split(test_targets, concurrency)[shard].tolist())
```

Comment:
- maybe don't use numpy? feels that it is too heavy to import numpy just for splitting an array.
- what will happen when the array is empty or has fewer than the number of shards?
- could you add a unit test?